Sources/SentryCrash/Recording/SentryCrash.m

```diff
@@ -40,7 +40,10 @@
 #import "SentryDefines.h"
 #import "SentryDependencyContainer.h"
 #import "SentryNSNotificationCenterWrapper.h"
+#import "SentrySDK+Private.h"
 #import <NSData+Sentry.h>
+#import <SentryClient+Private.h>
+#import <SentryHub.h>
```

Comment:
`h`: SentryCrash has a `sharedInstance` that initializes a default basePath in `getBasePath`. We need to make some changes so we use the basePath passed to `SentryCrashInstallation`.

Reply:
It looks like the `basePath` is changed in `-[SentryCrashInstallation install:]`. Should it be moved somewhere else?

Reply:
There's no longer a `+[SentryCrash sharedInstance]`, see now `SentryDependencyContainer.crashReporter` calling `[[SentryCrash alloc] init]` (you may need to merge our `main` branch again to see this, as we just recently merged #3247 after your reply).